src/annotator/test/cross-frame-test.js

```diff
@@ -3,10 +3,12 @@ import { CrossFrame, $imports } from '../cross-frame';
 describe('CrossFrame', () => {
   let fakeAnnotationSync;
   let fakeBridge;
+  let fakeHypothesisInjector;
   let fakeEventBus;
 
   let proxyAnnotationSync;
   let proxyBridge;
+  let proxyHypothesisInjector;
```

Comment:
The `proxyHypothesisInjector` name is consistent with the above, but these variables should really be called `Fake<ClassName>` to match our convention for naming fakes for classes and class instances.

Reply:
Resolved in the last commit.